adobe/react-spectrum
# RSP-1386 v3 Radio examples should include aria-label or FieldLabel for RadioGroup

Closes [RSP-1386](https://jira.corp.adobe.com/browse/RSP-1386)

## ✅ Pull Request Checklist:

- [x] Included link to corresponding issue [RSP-1386](https://jira.corp.adobe.com/browse/RSP-1386).
- Added/updated unit tests and storybook for this change (for new code or code which already has tests).

## 📝 Test Instructions:

1. Open Radio stories in Storybook starting with http://localhost:9003/?path=/story/radio--default
2. Verify that for each Radio story, the Accessibility plugin shows no errors due to insufficient labeling of the radio group.

## 🧢 Your Team:

Accessibility
